core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/RawlsApiService.scala

```diff
@@ -32,14 +33,20 @@ import scala.concurrent.{ExecutionContext, Future}
 import scala.concurrent.duration.FiniteDuration
 import scala.util.Try
 
-object RawlsApiService {
+object RawlsApiService extends LazyLogging {
   val exceptionHandler = {
     import org.broadinstitute.dsde.rawls.model.WorkspaceJsonSupport._
 
     ExceptionHandler {
       case withErrorReport: RawlsExceptionWithErrorReport =>
         complete(withErrorReport.errorReport.statusCode.getOrElse(StatusCodes.InternalServerError) -> withErrorReport.errorReport)
       case e: Throwable =>
+        // so we don't log the error twice when debug is enabled
+        if (logger.underlying.isDebugEnabled) {
+          logger.debug(e.getMessage, e)
```

Comment:
Nit: It looks the message logged here is still an `error` even when debug level is enabled? Can `logger.error(e.getMessage, e)` be used here too, or am I totally missing something?

Reply:
my point in this awkward construct was: if debug is enabled, log both the error message and the entire stack trace (many ugly lines in the logfile) by including the second argument `(e.getMessage, e)`. But if debug is not enabled, only log the single line of the message by only including one argument `(e.getMessage)`.